.github/dependabot.yml

```diff
@@ -18,6 +18,23 @@ updates:
         patterns:
           - "*"
 
+  # csi-isilon packages
+  - package-ecosystem: docker
```

Comment:
Changes should be merged into the release-v1.12.0 branch

Reply:
I've updated the the base branch to the release-v1.12.0 branch. I can create a separate PR for main

Reply:
Actually I already see you merged this section to main 2 weeks ago. Do I still need a PR for main?

Reply:
The csi-isilon configuration is not in main so need to be merged there so that dependabot whill check the files under /charts/csi-isilon

Reply:
Oh yes you're right. Misread the main branch file

Reply:
Main PR is here: https://github.com/dell/helm-charts/pull/491